src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.Sources.csproj

```diff
@@ -16,6 +17,10 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="README.md" Pack="true" PackagePath="/" />
```

Comment:
Choosing to use the same readme for both the WebJobs.Host and WebJobs.Host.Sources packages.